pkg/evaluators/metadata_singleflight_test.go

```
@@ -0,0 +1,193 @@
+		Method:          "GET",
+		AuthCredentials: auth.NewAuthCredential("", "authorization_header"),
+	}
+
+	cache := NewEvaluatorCache(&json.JSONValue{Static: "test-key"}, 60)
+
+	config := MetadataConfig{
+		Name:        "test-metadata",
+		Cache:       cache,
+		GenericHTTP: evaluator,
+	}
+	defer config.Clean(context.Background())
```

golangci-lint (2.12.2)

[error] Error return value of `config.Clean` is not checked

(errcheck)